### DISCUSSION TITLE
PostgreSQL DB: List of Databases not showing

### DISCUSSION THREAD

**Discussion by dhanan-np** [dbeaver/dbeaver#20700] [Q&A]
### Description

For a PostgreSQL DB, when connected using Admin user, the list of all the databases must be shown (output of this-> **SELECT datname FROM pg_database where datistemplate=false;**), from where I can select the appropriate database to work. 

### DBeaver Version

Community Edition 23.1.3

### Operating System

Windows

### Database and driver

PostgreSQL

### Steps to reproduce

1. Create connection of Admin for PostgreSQL.
2. Login using the Admin credential
3. Create a new Database 
4. refresh the connection. 
5. The newly created database doesn't appear in the list

It will not appear even if you remove the Database value from the Connection setting.

**Comment by LonwoLonwo** [comment]
Hello @dhanan-np 

Does this checkbox tick in your connection settings?

![2023-07-25 15_59_31-Connection _PostgreSQL - dvdrental_ configuration](https://github.com/dbeaver/dbeaver/assets/45152336/f4d168c9-58f6-4203-bdee-8fd8efa1d921)

**Comment by dhanan-np** [comment]
It is working now.
The other day I was trying with this option too and it was not working (I
restarted my machine)

**Comment by LonwoLonwo** [comment]
This option works only for databases with non-URL connection types. 
If your issue is solved now, I'm closing it.
